index.html

```diff
@@ -585,15 +585,15 @@ <h3>Use Cases and Requirements</h3>
 
       <section id="conformance" class="normative">
         <p>
-A <dfn>conforming document</dfn> is any concrete expression of the data model
-that complies with the normative statements in this specification.
-Specifically, all relevant normative statements in Sections
+A <dfn>conforming document</dfn> is a compact JSON-LD document that
```

Comment:
Is there an algorithm somewhere that verifiers can use to check that something is a compact JSON-LD document? I see https://w3c.github.io/json-ld-api/#compaction-algorithms defines how to compact a JSON-LD document: is the "is a compact JSON-LD document" predicate something like "https://w3c.github.io/json-ld-api/#compaction-algorithms returns a document that's identical to the original"?

Reply:
@jyasskin wrote:
> is the "is a compact JSON-LD document" predicate something like "https://w3c.github.io/json-ld-api/#compaction-algorithms returns a document that's identical to the original"?

Yes, exactly. 

@iherman wrote:
> there is a slight terminology mismatch, and we may want to use the term 'Compacted' rather than 'Compact'.

Fixed in 80bf0fadbd3a89a56eb2e8221badb48539b8254d